### ISSUE TITLE
July 2021 e-commerce VAT Package EU tax by shipping destination

### ISSUE THREAD

**Issue by mmtomm** [woocommerce/woocommerce#28720]
In July 2021 new tax regulations will take place within the european union - the so called 'e-commerce VAT Package'.
[more details at the EU website](https://ec.europa.eu/taxation_customs/business/vat/modernising-vat-cross-border-ecommerce_en)

the short version: physical goods will be treated like digital goods, the VAT rate of the destination country has to be applied instead of the origin country (when the treshold of €10K total net revenue for shipping to other EU countries is exceeded).

My question: how does this affect the woocommerce tax settings, will I be able to accomplish EU tax compliance with the existing functions in woocommerce and is there a 'how to' document available?

Thank you
Tom

**Comment by DavidAnderson684** [comment]
Since WooCommerce has been used successfully to sell digital goods since the start of 2015, when these regulations came in to treat digital goods with such place-of-supply rules, yes, it is already capable. Various extensions exist for helping with country-specific VAT/auditing/reporting issues (e.g. orders in one currency and reporting in another, need to retain evidence for place of supply, need to handle invoicing rules, etc.), e.g. https://wordpress.org/plugins/woocommerce-eu-vat-compliance/ , https://www.simbahosting.co.uk/s3/product/woocommerce-eu-vat-compliance/ (of which I am the author). (I'm not aware of any extension that will automatically alert you to change your tax rates when you're about to hit a threshold - AFAIK people currently handle that by manually monitoring their sales on a periodic basis).

N.B. Since this is the WooCommerce core issue tracker, if you have specific questions about extensions, you should go to the support sites for those.

**Comment by juliaamosova** [comment]
Hi @mmtomm,

Thank you for posting your question. We will take a look further at the upcoming change and provide a detailed answer after.

**Comment by EdithAllison** [comment]
To add: the €10000 threshold only applies within the EU. For non-EU businesses, there is no no threshold, a single sale is enough to trigger VAT liability and the need to register with the EU for VAT (non-union registration). But on the plus side, it's sufficient to register with 1 EU country (who will then process the VAT return & distribute the VAT to the other countries).

**Comment by mmtomm** [comment]
Here are some additional informations from the EU website about the 'EU VAT e-commerce rules' which come into effect in july 2021
[English PDF Explanatory Notes on VAT e-commerce rules](https://ec.europa.eu/taxation_customs/sites/taxation/files/vatecommerceexplanatory_28102020_en.pdf)
[German PDF Erläuterungen zu den Mehrwertsteuervorschriften für den elektronischen Geschäftsverkehr](https://ec.europa.eu/taxation_customs/sites/taxation/files/vatecommerceexplanatory_28102020_de.pdf)

**Comment by jonathan-dejong** [comment]
Hi @juliaamosova , 
Are there any updates on this and wether something needs to happen in WC core to accommodate for this change? 
In my opinion this is not extension territory if it affects all or most WC shops operating from the EU.

**Comment by DavidAnderson684** [comment]
As already stated, WooCommerce has no problems in buyer-location-based taxation. It's the "tax based on" setting, in combination with entering your tax tables correctly.

**Comment by MrMarci666** [comment]
Hi @DavidAnderson684 , that is true.

However, the EU is going to release a new VAT Package in July, 2021.
Here is the important part: You have to apply the origin VAT, until the net revenue to a country exceeds € 10k
At this point, from this moment, the next order should have the Destination VAT. This is like the dumbest idea in my opinion, but here we are. 👯 

I also think that this should be solved inside of WooCommerce, there has to be a built-in threshold, or something like that.

**Comment by mmtomm** [comment]
Hi,

another aspect which may need to be considered for this issue.

Let's face following circumstances:

Shop base is Austria (i.e. 20% tax)
Customer billing address is Germany (i.e. 19% tax)
Customer shipping address is Austria

Because the order does not leave Austria, a 20% VAT has to be applied although the billing address is Germany (Ziellandprinzip / destination country principle).
In this case, the revenue must not be recorded at the EU-OSS (one stop shop).
Since the EU-OSS requires a quarterly report (filing OSS) of revenues by country, such cases should be reflected (filtered out) in the woocommerce > reports > taxes ( or at least have columns with information like _billing destination country_ and _shipping destination country_) or something else.

**In short**: if the above order would have been shipped to Germany, a VAT of 19% had to be applied instead of 20% and the filing to the EU-OSS would be handled differently.

Thanks for reading
Tom

**Comment by vendidero** [comment]
Hi there,

just wanted to let you know that we've just released a compliance plugin for the One Stop Shop procedure specifically for WooCommerce. The plugin helps creating tax reports and (automatically) observe the delivery threshold. The plugin does furthermore allow you to add multiple tax classes per country per product, as some products might have different tax classes in different countries.

For your reference: https://wordpress.org/plugins/one-stop-shop-woocommerce/

The helper plugin will be part of [Germanized](https://wordpress.org/plugins/woocommerce-germanized/) in the future.

Cheers

**Comment by DavidAnderson684** [comment]
Our plugin "WooCommerce EU/UK VAT Compliance" at https://www.simbahosting.co.uk/s3/product/woocommerce-eu-vat-compliance/ has now added features to allow stores to comply with these requirements:

- Reports can now be displayed based on customer's taxation country, or upon the tax-receiving country (as noted by previous commenter, these are not necessarily the same; you need the former report if you want to see how near the threshold you are, and you need the latter to actually pay your taxes properly)
 
- A quick "year-to-date" report link has been added

- The shop owner can configure rules to switch product taxation classes when a threshold is met (this can be used for any VAT region and you can set the amount and currency as you please; for the new EU regulation you'd set it to 10,000 euros, and configure it to switch between a taxation class that is 'flat' (your base country's VAT) and one that varies on the customer's country). By using two separate tax classes (rather than changing the tax table entries dynamically in the background) I believe it is easier to audit afterwards and to control.

The plugin also now allows to multiple "VAT exemption based on value" rules for different countries, calculated in different currencies, in view of the place-of-supply regulations with the 150 euro cut-off that are coming in. (Up until now it only allow one rule, because the UK was the only country with such a rule).

The free version (https://wordpress.org/plugins/woocommerce-eu-vat-compliance/) includes the new reporting features so that shops can see when they are coming near to the threshold value. (Though it seems unlikely to me that anyone with 10,000 EUR of cross-border sales would not be wanting some of the other Premium features to aid their convenience).

**Comment by lucascysewski** [comment]
Hey guys!

Im happy you've been already working on this very important issue.

My concern is: we have a multisite on MLP + WooCommerce.
Let's say I've got 2 types of products:
- electronics (23% VAT in Poland)
- flowers (5% VAT in Poland)

I guess every and every site (polish, English, German) would need to be adjusted - so somehow we have to "match" the VAT. So if the customer wants to buy flowers with 5% VAT, it would need to adjust to 4% in Germany for example. The same for Electronics: 23% in Poland, 19% in Germany. 

Have you thought about it as well? How could we "match" 23% VAT from Poland with 19% VAT in Germany and others? Of course we will have to set It up on all multisites, but let's not worry about it yet ^_^"

Apart from that, can't we use the local VAT straightaway? Does it say we need to go through 10k limit VAT first? is it 10k/country or in general in EU?

Thank you for your effort!

**Comment by EdithAllison** [comment]
@lucascysewski All of this is already possible with default Woo. You set up different tax rates eg low / standard. Then you define the % rate for each of these rates for each country you sell to. And then you apply a rate (eg flowers = low / electornics = standard) to a product. Multisite doesn't pose any issues other than that you need to duplicate your settings across sites. For VAT registration this would be better answered by your accountant.

**Comment by TussendoorHQ** [comment]
Any news regarding this issue? I also think that this should be solved inside of WooCommerce, just like @MrMarci666 has mentioned.

**Comment by DavidAnderson684** [comment]
I don't speak for the WooCommerce team, but the possibility that we're likely to see tools for tracking when you get to sales over €10,000 to other countries within EU considered as core territory is basically zero. It's a specialised requirement for one part of the world, only relevant to shops that export a five-figure amount beyond their own national border. That's never been core territory, so realistically, anyone having to deal with these laws should not waste more than a few seconds hoping for it (N.B. There's an official ideas/requests board here: https://ideas.woocommerce.com/forums/133476-woocommerce).

**Comment by mmtomm** [comment]
There's another free plugin called [One Stop Shop for WooCommerce](https://wordpress.org/plugins/one-stop-shop-woocommerce/) from european based developers which can help you to monitor the treshold, switch from local vat to EU countries vat and have csv exports for the reporting in OSS (former MOSS)

**Comment by Maverick87Shaka** [comment]
WooCommerce still not support at all the ability to don't apply VAT for UK if you're an EU store, this is almost ridiculous.
And I'm even not sure why there is a Standard TAX rate table taht you can manually edit if it will be ignored during checkout.

**Comment by mmtomm** [comment]
@Maverick87Shaka I think you can accompish this by tax settings > based on > shipping destination and then setup a shipping class for UK with 0% tax.

**Comment by Maverick87Shaka** [comment]
> 

It's already set based on destination address, and set up a shipping class for GB doesn't have effects
